lib/MentionsForm/MentionsFormSearch.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown, Avatar, AvatarInformation } from '../../lib/';
+import { filterUsers } from './MentionsHelper';
+
+class MentionsFormSearch extends Component {
+  state = {
+    users: this.props.users
+  };
+
+  searchForUsers = e => {
+    if (e.target.value.trim() !== '' && this.props.users.length > 0) {
+      return this.setState({
+        users: filterUsers(this.props.users, e.target.value, true)
+      });
+    }
+    return this.setState({ users: this.props.users });
+  };
+
+  render() {
+    const users = this.state.users.map(user => (
+      <Dropdown.Action
+        action={() => this.props.addMention(user.id, user.display)}
+        key={user.id}
+        isSubmit={false}
+      >
+        <Avatar url={user.avatar} initials={user.initials}>
+          <AvatarInformation email={user.email} name={user.name} />
```

Comment:
Just realised we don't show the username here, is that a bug? And we don't show the email in the MentionsFormInput, only the username/name. They should be the same, right?

Reply:
Good point, I've added a prop which will determine whether both should display the email or username (although I'd imagine at some point we'll just end up displaying both ;))